### PR TITLE
Allign CI with rest of repos

### PR DESCRIPTION
**Pull request by Timple**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
no cpp here

**Comment by MatthijsBurgh** [comment]
@Rayman, Is this no problem for the way you compile the dashboard?

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - export PACKAGE=${TRAVIS_REPO_SLUG#*/}
+
 install:
-- npm install
-script: npm run build
-deploy:
-  provider: releases
-  api_key:
-    secure: W54Oa7xps+yMVd8wUQ/GCrgemcipZ8WTFL/lPGCZeLsqzA6hZ2CzevH/CigrZTmO0eQFIUfZKALJR7EzR9IbsSbIX4b6rgdpRZW6TX2bM0zIncX0JPyXxgnM2WIA5MiL2o7c00U83mqFyOd/BHEeSX9CP0TTL7rEa9HfWNxpo2U=
```
Comment (LoyVanBeek) [inline .travis.yml]: @Rayman when we put this back, making this a secret environment variable can't hurt, right?

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+services:
+  - docker
+
+before_install:
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - export PACKAGE=${TRAVIS_REPO_SLUG#*/}
+
 install:
-- npm install
```
Comment (LoyVanBeek) [inline .travis.yml]: I'm pretty sure this is in here for a reason. Is there a benefit from docker here even?

**Comment by Timple** [comment]
Lets not merge this, seems like a decent .travis.yml file was already present

File: .travis.yml (hunk 1 above)
Comment (Rayman) [inline .travis.yml]: This is already secret. Its encrypted. See the travis documentation

File: .travis.yml (hunk 2 above)
Comment (Rayman) [inline .travis.yml]: Yes, I already made a travis.yml

**Pull request closed without merging**